proposed/0002-release-schedule.md

```diff
@@ -0,0 +1,110 @@
+    is unclear then it is treated as MAJOR ("next") - OpenMage maintainers may also use discretion to override non-maintainers
+- The "next" branch should periodically be updated by merging the latest "main" branch - if urgent, any community member
+  may create a PR to merge the latest "main" or port a change into "next" from "main" and this should be accepted without hassle
+  - It is not the responsibility of the PR author of a MINOR patch to also maintain the "next" branch
+  - OpenMage maintainers may merge "main" into "next" without a PR
+- A newly tagged release on "main" bumps the MINOR version and no new branch is created
+- Just before a MAJOR release is tagged:
+  - Create a new branch from "main" named after the current latest MAJOR version number (e.g. v21 if last release was 21.x.x)
+  - Merge "main" into "next" and fast-forward "main" to "next" (so the two should have the exact same head)
+  - When a MAJOR release is tagged there should **not** also be a new MINOR release as well, the MAJOR release becomes the
+    very next version after the last MINOR release
+    - It is possible that to obtain some MINOR updates a user might be required to update to the newest release even if it
+      is a MAJOR version bump - or worst case they could specify a commit hash just before "next" was merged
+- Only significant security patches or regression fixes can be released immediately in a PATCH release; MINOR and MAJOR
+  updates should wait for the next release (e.g. there should not be two MINOR releases in the same month)
+  - Users who need features immediately after they are merged should be encouraged to use either "dev-main" or "dev-next"
+    and not ask maintainers for quicker releases
+- PATCH fixes will *only* be back-ported to MAJOR versions that are less than 2 years (730 days) old - this is the sole
+  purpose of the "v*" branches
+  - Backporting to old releases can be done by a maintainer directly on the appropriate branches or by the community via a PR
+  - The maintenance branches (v19, v20, etc.) may be deleted by any maintainer after they are no longer eligible to receive
+    PATCH updates - users should be discouraged from using these branches in production (e.g. `"dev-v19"`) - there is no reason
+    to do this because they can just use `"^19"` to always get the latest from the v19 branch until it is EOL at which point
+    they must update to a newer MAJOR version
+
+The "next" branch serves as a pre-release area for MAJOR changes in case there should be multiple MAJOR changes merged in
+a short period while allowing "main" to continue to be "stable" for users wanting only the latest MINOR updates. It is
+expected that most activity will occur in "main".
+
+#### Example
+
+As of now the current releases are 19.4.21 and 20.0.18. The next release containing only MINOR features would be 20.1.0, then 20.2.0
```

Comment:
I think we should limit ourselves to two subsets of numbers used in a version, for example 20.1, considering that we have 6 - 12 minor versions per year. The 3-set of number is no longer valid.

Reply:
We have previously not been using SemVer, but this RFC indicated that we should start. So MAJOR.MINOR.PATCH is needed.

Reply:
In this case I agree this format.